clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/LoadBalancerProvider.java

```diff
@@ -35,11 +35,11 @@
 public interface LoadBalancerProvider<T extends LoadBalancer> {
   String getCloudProvider();
 
-  List<Item> list();
```

Comment:
Only place the core code was changed was in this type, where we improved the generic signatures a bit.